src/utils/components/MatchExpression/Expression.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
```

Comment:
```suggestion
import React, { FC } from 'react';
```
Please remove all the `React.` usage and import react like this.


Console use to import react like this for a typescirpt issue that we don't have in this repo.